### PR TITLE
Update xccdf-resources-build.sh

### PR DESCRIPTION
**Pull request by Honny1**
This PR changes tools for build and minifies CSS and JS. Old tools didn't work or are not supported on last versions of Fedora.

### CODE REVIEW

**Comment by jan-cerny** [comment]
@Honny1 Welcome!

**Comment by jan-cerny** [comment]
@openscap-ci add to whitelist

**Comment by evgenyz** [comment]
Hey! That is a great start. But there are more problems with JS and CSS resources generation. The most important one is that they are not generated/complied/minified during the build, but reside in the repo in an almost binary form. Would you like to get on this as well?

**Comment by evgenyz** [comment]
Also there is (#1560) a nice-to-have upgrade of the Bootstrap.

**Comment by jan-cerny** [comment]
>  Hey! That is a great start. But there are more problems with JS and CSS resources generation. The most important one is that they are not generated/complied/minified during the build, but reside in the repo in an almost binary form. Would you like to get on this as well?

Great idea, but please in separate PRs.

File: xsl/xccdf-resources-build.sh [hunk 1]
```diff
@@ -1,6 +1,8 @@
-# A horrible little script that builds and minifies CSS and JS we bundle with HTML report and guide
+#!/bin/sh
+# This script builds and minifies CSS and JS that we bundle to HTML report and guide.
+set -e 
 
-for cmd in csstidy slimit; do
+for cmd in yuicompressor; do
```
Comment (jan-cerny) [inline xsl/xccdf-resources-build.sh:5]: You have only 1 item now, so the `for` cycle isn't necessary.

**Comment by jan-cerny** [comment]
I have tried to generate guide and report on Fedora OSPP profile and the generated documents look fine.

**Comment by evgenyz** [comment]
The one last thing maybe is to mention it in the docs as a build dep.

**Comment by jan-cerny** [comment]
In my opinion it isn't a build dependency because the rebuild isn't a part of build process. The csstidy and slimit also aren't mentioned. I wouldn't require to add it at this point.

**Pull request merged**